### PR TITLE
6.3.0 release merge to master

### PR DESCRIPTION
**Pull request by kyle-ciq**

### CODE REVIEW

File: FreestandingWPFExample/FreestandingWPFExample.csproj [hunk 1]
```diff
@@ -4,9 +4,7 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net5.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
-    <AssemblyVersion>6.2.0.0</AssemblyVersion>
-    <FileVersion>6.2.0.0</FileVersion>
-    <Version>6.2.0</Version>
+    <Platforms>AnyCPU;x64</Platforms>
```
Comment (NinjaManatee) [inline FreestandingWPFExample/FreestandingWPFExample.csproj:7]: It looks like some file version information was removed here.

File: WPFExampleCore/WPFExampleCore.csproj [hunk 2]
```diff
@@ -4,17 +4,17 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net5.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
-    <Version>6.2.0</Version>
+    <Platforms>AnyCPU;x64</Platforms>
```
Comment (NinjaManatee) [inline WPFExampleCore/WPFExampleCore.csproj:7]: Some version information was removed here.

File: WPFMultiWindowExampleCore/WPFMultiWindowExampleCore.csproj [hunk 3]
```diff
@@ -5,13 +5,13 @@
     <TargetFramework>net5.0-windows</TargetFramework>
     <UseWPF>true</UseWPF>
     <StartupObject>WPFMultiWindowExampleCore.AppStartup</StartupObject>
-    <Version>6.2.0</Version>
+    <Platforms>AnyCPU;x64</Platforms>
```
Comment (NinjaManatee) [inline WPFMultiWindowExampleCore/WPFMultiWindowExampleCore.csproj:8]: Some version information was lost here.

File: WindowlessExample.Core/WindowlessExample.Core.csproj [hunk 4]
```diff
@@ -1,14 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
-    <Version>6.2.0</Version>
+    <Platforms>AnyCPU;x64</Platforms>
```
Comment (NinjaManatee) [inline WindowlessExample.Core/WindowlessExample.Core.csproj:6]: Some version information was lost here

File: WinformExampleCore/WinformExampleCore.csproj [hunk 5]
```diff
@@ -4,13 +4,13 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net5.0-windows</TargetFramework>
     <UseWindowsForms>true</UseWindowsForms>
-    <Version>6.2.0</Version>
+    <Platforms>AnyCPU;x64</Platforms>
```
Comment (NinjaManatee) [inline WinformExampleCore/WinformExampleCore.csproj:7]: Some version information was lost here

**Review by NinjaManatee — APPROVED** [review]
Looks good but for losing version information in a few places.

**Review by MaksimDedov — APPROVED** [review]
Looks good, except missed versions

**Review by NinjaManatee — APPROVED** [review]

**Pull request merged**